build-tests-samples/heft-storybook-react-tutorial/package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "build": "heft build --clean --storybook",
```

Comment:
Removing the `--storybook parameter`? Will this still test Storybook during the build?

Reply:
We could also update the Rush-level command definitions to add an optional `_phase:storybook` to projects, and have the command that includes those be the one we run on the CI builds.

Reply:
would it be an idea to defer that to a later update or would this be blocking?

Reply:
I don't think this demo is significant enough to justify introducing a storybook phase for the entire Rush Stack monorepo.

The importance of including the storybook output in CI builds is to ensure that it won't get accidentally broken.

I think we can accomplish that like this:
```js
    "build": "heft build --clean",
    . . .
    "build-storybook": "heft build --storybook",
    "_phase:build": "heft run --only build -- --clean --storybook", 👈👈
```

In this way, the `rush build` command will validate both operations, but the `rushx build` command will still illustrate your recommended usage.

Reply:
> I think we can accomplish that like this:

I pushed this change to your branch. Otherwise this PR looks good to go. 👍